src/modules/fancyzones/editor/FancyZonesEditor/StringUtils.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+
+namespace FancyZonesEditor.Utils
+{
+    public static class StringUtils
+    {
+        public static string UpperCamelCaseToDashCase(this string str)
+        {
+            // If it's single letter variable, leave it as it is
+            if (str.Length == 1)
+            {
+                return str;
```

Comment:
Not sure, but I guess we need to call `ToLower()` here ?

Reply:
We use upper case for `X` and `Y` coords for example. That would require bigger changes in C++ code as well . I would prefer to keep it is. At least for now.

Reply:
Oh, in that case, we can leave it.